committeeoversightapp/fixtures/hearingcategorytype.json

```diff
@@ -0,0 +1,58 @@
+[
```

Comment:
Can you tell me a bit about the purpose of the JSON file?

Reply:
Lugar has 8 categories of hearing, and they'd like to assign one to each hearing as it's entered. I talked to Forest and these don't fit well into the OCD event model, so I've made two new models to house the categories and their association with hearings: HearingCategory and HearingCategory type. The categories should be a pre-populated drop down menu in the form, and this fixture lets me load the data into the app.